rtnetlink/src/addr/add.rs

```diff
@@ -70,6 +70,10 @@ impl AddressAddRequest {
         AddressAddRequest { handle, message }
     }
 
+    pub(crate) fn new_raw(handle: Handle, message: AddressMessage) -> Self {
```

Comment:
I think it would be nice to document why someone would want to use the `_raw` methods. I'm myself not too sure about it.

Reply:
I mean I understand that you may need to tweak the messages sometimes. But in `LinkRequest` we have [`message_mut()`](https://github.com/little-dude/netlink/blob/3cf41bf7dd658abfdb07e1dd0c52bfaf5035e8f7/rtnetlink/src/link/add.rs#L298) for this. Could we use that here too, for the sake of consistency?

Reply:
Sure. I will use the `message_mut()` way. Thanks.